docker/deploy-fedimintd/deploy.sh

```diff
@@ -106,10 +106,13 @@ ssh -q "root@$ssh_host" << EOF
 
   sed -i 's/my-super-host.com/$domain/g' $host_dir/.env
 
+  ufw allow 8173/tcp
+  ufw allow 8174/tcp
   ufw allow 80/tcp
   ufw allow 443/tcp
   ufw allow 22/tcp
   ufw --force enable
+  ufw default deny
```

Comment:
Not a big deal for our docker env, but in general would it be best practice to first define `ufw default deny`, then add `allow` then `enable`?

Reply:
I would always put allows first, just to not risk locking out, but I I guess `enable` should go last indeed.